examples/example-runner/Cargo.toml

```diff
@@ -8,18 +8,16 @@ license = "MIT OR Apache-2.0"
 # See rustc_codegen_spirv/Cargo.toml for details on these features
 [features]
 default = ["use-compiled-tools"]
-use-installed-tools = ["spirv-builder/use-installed-tools"]
-use-compiled-tools = ["spirv-builder/use-compiled-tools"]
+use-installed-tools = ["example-shader/use-installed-tools"]
+use-compiled-tools = ["example-shader/use-compiled-tools"]
```

Comment:
I'm not really a fan of having to propagate these features, but I don't see a different way at the moment.